distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc

```diff
@@ -1,26 +1,533 @@
+|
+|
+
+|URL Property Key
+|Property that the accessURI value should be put into for source creation
+|
+|
+
+4+|
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|===
+
+.Organizations Tab (click *Add* to add an organization)
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Organization Name
+|This organization's name
+|string
+|yes
+
+|Address
+|This organization's primary address
+|Expand to enter address information
+|yes
+
+|TelephoneNumber
+|||
+
+|Email
+|||
+
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|===
+
+.Contacts (click *Add* button to add contact info)
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Contact Title
+|Contact Title
+|String
+|yes
+
+|Contact First Name
+|Contact First Name
+|String
+|yes
+
+|Contact Last Name
+|Contact Last Name
+|String
+|yes
+
+|Address
+|Address for listed contact
+|String
+|minimum one
+
+|Phone number
+|Contact phone number
+|
+|minimum one
+
+|Email
+|Contact email
+|String
+|minimum one
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|===
+
+.Collections (Click *Add* to add Content Collection(s))
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Content Name
+|Name for this metadata content
+|string
+|yes
+
+|Content Description
+|Short description for this metadata content
+|string
+|no
+
+|Content Object Type
+|The kind of content object this will be. Default value should be used in most cases.
+|string
+|yes
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|===
+
+===== [[adding_a_service_binding]]Adding a Service Binding
+
+* Navigate to ${admin-console}.
+* Select ${ddf-registry} or ${ddf-catalog}.
+** (*Local Registry Nodes* tab is editable from either application.)
+* Click the name of the desired node.
+* Enter (at minimum) a *Node Name* under *General Information* tab).
+* Click the *Services* tab.
+* Click *Add* to add a service.
+* Expand new Service.
+* Enter Service name and details.
+* Click *Add* to add binding.
+* Select Service Binding type.
+** Select one of the defaults or _empty_ for a custom service binding.
+** If selecting _empty_, fill in all required fields.
+* Click Save.
 
 ==== Publishing to Others
 
-===== Editing Publishing Settings
+* Navigate to the *Remote Registries* tab in either ${ddf-registry} or ${ddf-catalog} application.
+* Click *Add* to add a remote registry.
+* Give node a unique local name.
+* Enter Registry Service (CSW) Url.
+* Confirm Allow Publish is checked.
```

Comment:
I was under the impression that we were sticking with the Push/Pull terminology, but I also know that some of those details have been in discussion recently. For now it is "Push".